### PR TITLE
Add package: aff-cache

### PR DESCRIPTION
**Pull request by jvliwanag**

### CODE REVIEW

File: new-packages.json [hunk 1]
```diff
@@ -47,5 +47,6 @@
   "purescript-point-free": "https://github.com/ursi/purescript-point-free.git",
   "purescript-parsing-hexadecimal": "https://github.com/markfarrell/purescript-parsing-hexadecimal.git",
   "purescript-ffi-foreign": "https://github.com/markfarrell/purescript-ffi-foreign.git",
-  "purescript-protobuf": "https://github.com/xc-jp/purescript-protobuf.git"
+  "purescript-protobuf": "https://github.com/xc-jp/purescript-protobuf.git",
+  "purescript-untagged-union": "https://github.com/jvliwanag/purescript-aff-cache.git"
```
Comment (f-f) [inline new-packages.json]:
```suggestion
  "purescript-aff-cache": "https://github.com/jvliwanag/purescript-aff-cache.git"
```

**Review by f-f — CHANGES_REQUESTED** [review]
It's most likely a typo, but if it's not I appreciate the pentesting effort :smile:

File: new-packages.json (hunk 1 above)
Comment (jvliwanag) [reply new-packages.json]: 😰

**Review by f-f — APPROVED** [review]

**Comment by jvliwanag** [comment]
Thanks!

**Comment by f-f** [comment]
You're welcome 🙂

**Pull request merged**